src/video_core/renderer_opengl/gl_resource_manager.h

```diff
@@ -1,10 +1,11 @@
-// Copyright 2022 Citra Emulator Project
+// Copyright Citra Emulator Project / Azahar Emulator Project
 // Licensed under GPLv2 or any later version
 // Refer to the license.txt file included.
 
 #pragma once
 
 #include <span>
+#include <string_view>
```

Comment:
Is this used?

Reply:
Yes, it was not failing before because this file was being included after the microprofile header.